lib/core/src/lib/form/components/widgets/date-time/date-time.widget.html

```diff
@@ -1,36 +1,49 @@
+        <mat-form-field class="adf-date-time-widget"
+                        [class.adf-left-label-input-datepicker]="field.leftLabels"
+                        [hideRequiredMarker]="true">
+            <label class="adf-label" *ngIf="!field.leftLabels" [attr.for]="field.id">
+                {{ field.name | translate }} ({{ field.dateDisplayFormat }})
+                <span class="adf-asterisk" *ngIf="isRequired()">*</span>
+            </label>
             <input matInput
-                [matDatetimepicker]="datetimePicker"
-                [id]="field.id"
-                [(ngModel)]="value"
-                [required]="isRequired()"
-                [disabled]="field.readOnly"
-                (change)="onValueChanged($event)"
-                (dateChange)="onDateChanged($event)"
-                (keydown.enter)="datetimePicker.open()"
-                [placeholder]="field.placeholder"
-                [matTooltip]="field.tooltip"
-                (blur)="markAsTouched()"
-                matTooltipPosition="above"
-                matTooltipShowDelay="1000"
-                [min]="minDate"
-                [max]="maxDate">
-            <mat-datetimepicker-toggle matSuffix [for]="datetimePicker" [disabled]="field.readOnly"></mat-datetimepicker-toggle>
+                   [matDatetimepicker]="datetimePicker"
+                   [id]="field.id"
+                   [(ngModel)]="value"
+                   [required]="isRequired()"
+                   [disabled]="field.readOnly"
+                   (change)="onValueChanged($event)"
+                   (dateChange)="onDateChanged($event)"
+                   [placeholder]="field.placeholder"
+                   [matTooltip]="field.tooltip"
+                   (blur)="markAsTouched()"
+                   matTooltipPosition="above"
+                   matTooltipShowDelay="{{ 1000 }}"
+                   [min]="minDate"
+                   [max]="maxDate">
+            <mat-datetimepicker-toggle matSuffix [for]="datetimePicker"
+                                       [disabled]="field.readOnly"></mat-datetimepicker-toggle>
         </mat-form-field>
         <error-widget [error]="field.validationSummary"></error-widget>
-        <error-widget *ngIf="isInvalidFieldRequired() && isTouched()" required="{{ 'FORM.FIELD.REQUIRED' | translate }}"></error-widget>
+        <error-widget *ngIf="isInvalidFieldRequired() && isTouched()"
+                      required="{{ 'FORM.FIELD.REQUIRED' | translate }}"></error-widget>
         <mat-datetimepicker #datetimePicker
-            data-automation-id="adf-date-time-widget-picker"
-            type="datetime"
-            [touchUi]="true"
-            [timeInterval]="5"
-            [disabled]="field.readOnly">
+                            data-automation-id="adf-date-time-widget-picker"
+                            type="datetime"
+                            [touchUi]="true"
+                            [openOnFocus]="true"
```

Comment:
openOnFocus was not present before. Can you please confirm if this addition is intentional?

Reply:
that was removed last month, that is a merging mistake 🙈